pyteamcity.py

```diff
@@ -105,8 +105,8 @@ def get_all_plugins(self):
         the Client.
         """
 
-    @GET('builds/?start={start}&count={count}')
-    def get_all_builds(self, start=0, count=100):
+    @GET('builds/?start={start}&count={count}&locator=branch:{branch}')
+    def get_all_builds(self, start=0, count=100, branch=''):
```

Comment:
There used to be a `set_branch()` method on the original -- which understandably was not great since it mutated the client in place (not thread-safe). But just a note that we might want to support everything the original client had implemented as `.set_*` methods eventually.